<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="FASTAFile">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPlainTextEdit" name="fBOX">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>571</width>
        <height>471</height>
       </rect>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
      <property name="centerOnScroll">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Your Fasta File is Empty...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>25</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>FASTA File</string>
      </property>
     </widget>
     <widget class="QLabel" name="Path_label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTextEdit" name="AlignmentBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>1071</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QTreeWidget" name="nameList">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>256</width>
        <height>391</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string>Sequence Names: </string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameBox">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Gene:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vGene">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>180</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dGene">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>210</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="jGene">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V Gene</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D Gene</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>J Gene</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>210</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D Gene</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dLocus">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>210</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>150</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Locus:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>180</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V Gene</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>240</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>J Gene</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="jLocus">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>240</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vLocus">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>180</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CDR3:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>370</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Peptide</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="peptide">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>370</y>
        <width>391</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DNA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>400</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CDR_len">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>400</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CDR_DNA">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>340</y>
        <width>391</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Somatic Mutations:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Som_mut">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>90</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Isotype:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="isobox">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>460</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Clonality:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>490</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Clone ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clone_id">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>490</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>499</x>
        <y>491</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clone_count">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>490</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>None</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTextEdit" name="igblastOut">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>821</width>
        <height>491</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QListWidget" name="tree_select">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>256</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Phylogenetic Trees:</string>
      </property>
     </widget>
     <widget class="QLabel" name="graphics_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>671</width>
        <height>551</height>
       </rect>
      </property>
      <property name="text">
       <string>image:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ExportData"/>
    <addaction name="ExportSelectedData"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuClonality">
    <property name="title">
     <string>Clonality</string>
    </property>
    <addaction name="Show_Distrobution_Plot"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuClonality"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>\</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="Show_Distrobution_Plot">
   <property name="text">
    <string>Show Distrobution Plot</string>
   </property>
  </action>
  <action name="ExportData">
   <property name="text">
    <string>Export Data</string>
   </property>
  </action>
  <action name="ExportSelectedData">
   <property name="text">
    <string>Export Selected Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
